src/frontend/komponenter/Fagsak/Saksoversikt/utlis.tsx

```diff
@@ -0,0 +1,207 @@
+import type { ReactNode } from 'react';
+import React from 'react';
+
+import { ExternalLink } from '@navikt/ds-icons';
+import { Link } from '@navikt/ds-react';
+
+import {
+    behandlingsresultater,
+    BehandlingStatus,
+    behandlingstyper,
+    behandlingÅrsak,
+    erBehandlingHenlagt,
+} from '../../../typer/behandling';
+import type { IBehandlingstema } from '../../../typer/behandlingstema';
+import { tilBehandlingstema } from '../../../typer/behandlingstema';
+import type { IMinimalFagsak } from '../../../typer/fagsak';
+import type { IKlagebehandling } from '../../../typer/klage';
+import { Klagebehandlingstype } from '../../../typer/klage';
+import type { ITilbakekrevingsbehandling } from '../../../typer/tilbakekrevingsbehandling';
+import {
+    Behandlingsresultatstype,
+    Tilbakekrevingsbehandlingstype,
+} from '../../../typer/tilbakekrevingsbehandling';
+import type { VisningBehandling } from './visningBehandling';
+
+export enum Saksoversiktstype {
```

Comment:
Burde denne heller hete noe ala SaksoversiktBehandlingstype? For saksoversikten er vel hele siden, men denne enumen skal representere hver av behandlingene på saksoversikten?

Reply:
[Saksoversiktstype -> Saksoversiktsbehandling](https://github.com/navikt/familie-ba-sak-frontend/pull/2467/commits/1c405b29f35987cb9d65cfc75bbc6435608e9df6)